From fcecfcb9c3bfe3b07c4d0a5be2d56a00d037d2ec Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 26 Nov 2018 16:10:05 +0900
Subject: [PATCH 701/909] arm64: dts: r8a77990: Detach IPMMU-VI0/1 from FCPVD

commit ce383706d55968c21741e7b9dadc4e402f6ad83d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch changes the IPMMU address translation for FCPVD to disabled
by default. This maintains the same IPMMU support environment
as the previously released R-Car Linux BSP.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 2 --
 arch/arm64/boot/dts/renesas/ulcb.dtsi     | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index 8defc25179cd..1e674843e70d 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -1705,7 +1705,6 @@
 			clocks = <&cpg CPG_MOD 603>;
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 603>;
-			iommus = <&ipmmu_vi0 8>;
 		};
 
 		fcpvd1: fcp@fea2f000 {
@@ -1714,7 +1713,6 @@
 			clocks = <&cpg CPG_MOD 602>;
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 602>;
-			iommus = <&ipmmu_vi0 9>;
 		};
 
 		vsp@fe960000 {
diff --git a/arch/arm64/boot/dts/renesas/ulcb.dtsi b/arch/arm64/boot/dts/renesas/ulcb.dtsi
index 94d58a8da31f..b60fd45f0a17 100644
--- a/arch/arm64/boot/dts/renesas/ulcb.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb.dtsi
@@ -32,7 +32,7 @@
 		clock-frequency = <12288000>;
 	};
 
-	avb-mch {
+	avb-mch@ec5a0100 {
 		compatible = "renesas,avb-mch-gen3";
 		reg = <0 0xec5a0100 0 0x100>; /* ADG_AVB */
 		reg-name = "adg_avb";
-- 
2.17.1

